js/tic_tac_toe.js

```diff
@@ -1,2 +1,140 @@
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box4.attr('class') === 'box_cell black' && $box5.attr('class') === 'box_cell black' && $box6.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //row 3
+    else if ($box7.attr('class') === 'box_cell red' && $box8.attr('class') === 'box_cell red' && $box9.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box7.attr('class') === 'box_cell black' && $box8.attr('class') === 'box_cell black' && $box9.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //col 1
+    else if ($box1.attr('class') === 'box_cell red' && $box4.attr('class') === 'box_cell red' && $box7.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box1.attr('class') === 'box_cell black' && $box4.attr('class') === 'box_cell black' && $box7.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //col 2
+    else if ($box2.attr('class') === 'box_cell red' && $box5.attr('class') === 'box_cell red' && $box8.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box2.attr('class') === 'box_cell black' && $box5.attr('class') === 'box_cell black' && $box8.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //col 3
+    else if ($box3.attr('class') === 'box_cell red' && $box6.attr('class') === 'box_cell red' && $box9.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box3.attr('class') === 'box_cell black' && $box6.attr('class') === 'box_cell black' && $box9.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //diagonal right
+    else if ($box1.attr('class') === 'box_cell red' && $box5.attr('class') === 'box_cell red' && $box9.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box1.attr('class') === 'box_cell black' && $box5.attr('class') === 'box_cell black' && $box9.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+
+    //diagonal left
+    else if ($box3.attr('class') === 'box_cell red' && $box5.attr('class') === 'box_cell red' && $box7.attr('class') === 'box_cell red'){
+      alert('red is the winnah!');
+      location.reload();
+    }
+    else if ($box3.attr('class') === 'box_cell black' && $box5.attr('class') === 'box_cell black' && $box7.attr('class') === 'box_cell black'){
+      alert('black is the winnah!');
+      location.reload();
+    }
+    else if ($('.red').length === 5) {
+      alert('tie');
```

Comment:
I'd avoid use of alerts if possible. Better to put something else on the page and have a reload button. But this works!